src/legacy/core_plugins/kibana/public/visualize/listing/visualize_listing_table.js

```diff
@@ -167,183 +194,29 @@ class VisualizeListingTableUi extends Component {
     return icon;
   }
 
-  sortByTitle = () => this.sortOn('title');
-  sortByType = () => this.sortOn('type');
-
-  renderHeader() {
-    return [
-      {
-        content: 'Title',
-        onSort: this.sortByTitle,
-        isSorted: this.state.sortedColumn === 'title',
-        isSortAscending: this.sortableProperties.isAscendingByName('title'),
-      },
-      {
-        content: 'Type',
-        onSort: this.sortByType,
-        isSorted: this.state.sortedColumn === 'type',
-        isSortAscending: this.sortableProperties.isAscendingByName('type'),
-      },
-    ];
-  }
-
-  renderRowCells(item) {
-
+  renderFlaskIcon(item) {
     let flaskHolder;
     if (item.type.shouldMarkAsExperimentalInUI()) {
-      flaskHolder = <span className="kuiIcon fa-flask ng-scope">&nbsp;</span>;
+      flaskHolder =  (
+        <EuiIcon
+          className="visListingTable__typeIcon"
+          aria-hidden="true"
+          type="beaker"
+          size="m"
+        />
+      );
     }else{
       flaskHolder = <span />;
```

Comment:
This isn't necessary - everything falsy just renders as nothing in React - the whole function can be simplified to
```
  renderFlaskIcon(item) {
    return item.type.shouldMarkAsExperimentalInUI() && (
         <EuiIcon
          className="visListingTable__typeIcon"
          aria-hidden="true"
          type="beaker"
          size="m"
        />
    );
}
```

Reply:
done